### DISCUSSION TITLE
IETF discussion about internet and it's future - please join!

### DISCUSSION THREAD

**Discussion by dumblob** [dswd/vpncloud#210] [Announcements]
https://mailarchive.ietf.org/arch/msg/ietf/-Sn9nihA4Gc9bZXODdxRtQuz-Bg/

Please join the mailing list discussion as vpncloud kind of solves (by "workarounds" but anyway) some/many of the raised issues in the mailing list thread.

Thanks!

**Comment by dswd** [comment]
Hi there,

could you fill me in on how VpnCloud is related to that discussion?

**Reply by dumblob** [reply]
I see vpncloud as a middle-step between current networking and CCNs (content-centric networks). The discussion touches on (some of) the issues CCNs try to solve, so I thought the community around vpncloud could be interested in that as well.
